### PR TITLE
Adds stop_all_apps action

### PR DESCRIPTION
**Pull request by singhgarima**
Signed-off-by: Garima Singh <igarimasingh@gmail.com>

Notes:
I have also made a small change to the README on how to run tests. 
from `pytest` to `python setup.py test`
The reason for this change is:
* We already had the logic in setup.py to support pytest-runner. (I added the same as a dependency in requirements-dev.txt)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -177,5 +177,5 @@ environment, even when running from the `chaos` command locally.
 To run the tests for the project execute the following:
 
 ```
-$ pytest
+$ python setup.py test
```
Comment (Lawouach) [inline README.md:180]: Interested to know why this changed? :)

File: chaoscf/actions.py [hunk 2]
```diff
@@ -6,11 +6,12 @@
 from logzero import logger
 
 from chaoscf.api import call_api, get_app_by_name, get_app_routes_by_host, \
-    get_app_instances, get_bind_by_name, get_routes_by_host
+    get_app_instances, get_bind_by_name, get_routes_by_host, get_apps_for_org
```
Comment (Lawouach) [inline chaoscf/actions.py:9]: Do you mind to sort the imports alphabetically? It makes a bigger commit change but it's easier to read the imports afterwards.

File: chaoscf/actions.py [hunk 3]
```diff
@@ -6,11 +6,12 @@
...
 
 __all__ = ["delete_app", "remove_routes_from_app", "terminate_app_instance",
            "terminate_some_random_instance", "unbind_service_from_app",
-           "unmap_route_from_app", "map_route_to_app", "stop_app"]
+           "unmap_route_from_app", "map_route_to_app", "stop_app",
+           "stop_all_apps"]
```
Comment (Lawouach) [inline chaoscf/actions.py]: Similar idea here :)

File: chaoscf/api.py [hunk 4]
```diff
@@ -1,16 +1,16 @@
 # -*- coding: utf-8 -*-
 from typing import Any, Dict, List
 
+import requests
 from chaoslib.exceptions import FailedActivity
 from chaoslib.types import Configuration, Secrets
 from logzero import logger
-import requests
```
Comment (Lawouach) [inline chaoscf/api.py]: We prefer keeping imports sorted alphabatically.

File: chaoscf/api.py [hunk 5]
```diff
@@ -1,16 +1,16 @@
...
 
 from chaoscf import auth
 
 __all__ = ['call_api', 'get_app_by_name', 'get_org_by_name',
            'get_space_by_name', 'get_app_instances', 'get_app_routes_by_host',
-           'get_routes_by_host', 'get_bind_by_name']
+           'get_routes_by_host', 'get_bind_by_name', 'get_apps_for_org']
```
Comment (Lawouach) [inline chaoscf/api.py]: same :)

File: tests/test_actions.py [hunk 6]
```diff
@@ -1,21 +1,21 @@
 # -*- coding: utf-8 -*-
-from unittest.mock import patch
+from unittest.mock import patch, call
```
Comment (Lawouach) [inline tests/test_actions.py]: same :p

**Review by Lawouach — CHANGES_REQUESTED** [review]
LGTM

I think the feature looks great, just nitpicking about Python's linting here and there.

File: README.md (hunk 1 above)
Comment (singhgarima) [reply README.md:180]: I mentioned in the notes for PR :)

File: chaoscf/actions.py (hunk 3 above)
Comment (singhgarima) [reply chaoscf/actions.py]: sure :)

File: chaoscf/actions.py (hunk 2 above)
Comment (singhgarima) [reply chaoscf/actions.py:9]: Done!

File: chaoscf/actions.py (hunk 3 above)
Comment (singhgarima) [reply chaoscf/actions.py]: Done!

File: chaoscf/api.py (hunk 4 above)
Comment (singhgarima) [reply chaoscf/api.py]: Done!

File: tests/test_actions.py (hunk 6 above)
Comment (singhgarima) [reply tests/test_actions.py]: Done!

File: chaoscf/api.py (hunk 5 above)
Comment (singhgarima) [reply chaoscf/api.py]: Done!

File: README.md (hunk 1 above)
Comment (Lawouach) [reply README.md:180]: Gotcha! I missed that bit :p

**Review by Lawouach — APPROVED** [review]

**Pull request merged**